congress_lookup.py

```diff
@@ -0,0 +1,110 @@
+        parser.add_argument('-D', '--download', help="Download data", action='store_true', default=True)
+        parser.add_argument('-g', '--debug', help="Debug flag", action='store_true')
+        return parser.parse_args()
+
+    def lookup_property(self,property):
+        if self.args.committee is not None:
+            self.lookup_by_committee(property)
+        if self.args.last_name is not None:
+            self.lookup_by_lastname(property)
+
+    def lookup_by_committee(self,property):
+        for comm in (comm for comm in self.committees if self.inclusive_wildcard_match(comm['name'],self.args.committee)):
+            if self.args.debug: print(comm)
+            print('"{}" member properties:'.format(comm['name'].encode('utf-8')))
+            members = self.membership[comm['thomas_id']] if comm['thomas_id'] in self.membership else []
+            for member in members: self.lookup_by_member(property,member)
+
+    def inclusive_wildcard_match(self,name,pat):
+        if any(c in pat for c in '*?[]'):	# a wildcard pattern
+            # prepend or append a * for inclusiveness if not already there
+            if pat[0] is not '*': pat = '*' + pat
+            if pat[-1] is not '*': pat = pat + '*'
+        else:					# not a wildcard
+            pat = '*' + pat + '*'
+        return fnmatch.fnmatch(name,pat)
+
+    def lookup_by_member(self,property,member):
+        for leg in ( leg for leg in self.legislators if \
+                    (leg['name']['official_full'] == member['name']) \
+                    or ('bioguide' in leg['id'] and leg['id']['bioguide'] == member['bioguide']) \
+                    or ('thomas' in leg['id'] and leg['id']['thomas'] == member['thomas']) ):
+            self.lookup_legislator_properties(property,leg)
+
+    def lookup_by_lastname(self,property):
+        for leg in (leg for leg in self.legislators if fnmatch.fnmatch(leg['name']['last'],self.args.last_name)):
+            if self.args.debug: print(leg)
+            self.lookup_legislator_properties(property,leg)
+
+    def lookup_legislator_properties(self,property,legislator):
+        self.properties[property] = [term[property] for term in legislator['terms'] if property in term and len(term[property]) > 0]
+        for off in self.offices:
+            if self.args.debug: print(off)
+            if any(off['id'][db] == legislator['id'][db] for db in off['id'] if db in off['id'] and db in legislator['id']):
+                self.properties[property] += [ok[property] for ok in off['offices'] if property in ok and len(ok[property]) > 0]
+                break
+        print('Property \'{}\' for {}:'.format(property,legislator['name']['official_full'].encode('utf-8')))
+        print('\n'.join(self.properties[property]))
+
+    def database_load(self):
+        try:
+	    with self.database_access('legislators-current.yaml') as y:
```

Comment:
There's a `pyflakes` error [breaking the build](https://travis-ci.org/unitedstates/congress-legislators/builds/197515108) on this line:

```
./congress_lookup.py:84:65: inconsistent use of tabs and spaces in indentation
	    with self.database_access('legislators-current.yaml') as y:
```